tests/integration/cairo_path/cairo_path_test.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+import pytest
+
+from protostar.cairo.cairo_bindings import (
+    call_test_collector,
+    call_protostar_sierra_to_casm,
+    call_cairo_to_casm_compiler,
+    call_cairo_to_sierra_compiler,
+    call_sierra_to_casm_compiler,
+    call_starknet_contract_compiler,
+)
+
+from tests.integration.conftest import CreateProtostarProjectFixture
+from tests.integration.cairo_compiler.prepare_files_fixture import (
+    PrepareFilesFixture,
+    RequestedFile,
+)
+
+
+@pytest.fixture(name="prepare_files")
+def prepare_files_fixture(create_protostar_project: CreateProtostarProjectFixture):
+    with create_protostar_project() as protostar:
+        protostar.build_sync()
+        return PrepareFilesFixture(protostar)
+
+
+SCRIPT_ROOT = Path(__file__).parent
+CONTRACTS_DIR = SCRIPT_ROOT / "contracts"
+
+
+def test_cairo_path_for_starknet_contract():
+    casm_contents = call_starknet_contract_compiler(
+        input_path=CONTRACTS_DIR / "starknet_project" / "basic_starknet_contract.cairo",
+        cairo_paths=[CONTRACTS_DIR / "external_lib"],
+    )
+    assert casm_contents
+
+    with pytest.raises(Exception):
+        call_starknet_contract_compiler(
+            input_path=CONTRACTS_DIR
+            / "starknet_project"
+            / "basic_starknet_contract.cairo",
+        )
+
+
+def test_cairo_path_for_starknet_test(prepare_files: PrepareFilesFixture):
+    prepared_files = prepare_files.prepare_files(
```

Comment:
The name `PrepareFilesFixture` could be more meaningful.

Reply:
1. You never proposed any alternative so I don't know what you mean.
2. It is a fixture which prepares files for tests, the name is pretty descriptive to me.